brave/brave-core
# Fix tab audio muting button

Resolves https://github.com/brave/brave-browser/issues/21829

Chromium used to support its own tab muting button but that was removed in 2018. Since then, we added our own button and a flag (defaulting to enabled) to control it. In CR100, Chromium reimplemented tab muting via https://chromium-review.googlesource.com/c/chromium/src/+/3123736.

In order to account for this, I removed our implementation of the tab muting button and our Brave-specific flag. We now use Chromium's implementation of the button and their flag (brave://flags/#enable-tab-audio-muting).

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Visit https://www.youtube.com/watch?v=NSZ-D8YkodI
- Play the video
- Verify that pressing the audio mute icon mutes the video
- Verify that pressing the audio mute icon again unmutes the video
- Visit brave://flags/#enable-tab-audio-muting
- Set feature to disabled
- Relaunch browser
- Visit https://www.youtube.com/watch?v=NSZ-D8YkodI
- Play the video
- Verify that audio mute icon isn't interactive
